src/index.js

```diff
@@ -160,3 +168,18 @@ export function register(ReactComponent, tagName, mapping = {}) {
 
   return customElements.define(tagName, WebReactComponent);
 }
+
+// register is default export
+export default register;
+
+// additionally everything is exported as named export for convenience
+export {
+  register,
```

Comment:
KISS is good! Refactored, but added `Types` and `register` as a named exports. The default export is an Object containing both as a property, like Preact does it: https://github.com/developit/preact/blob/master/src/preact.js

So you can use it like:
```javascript
import WebReactComponent, { register, Types } from 'web-react-component';

console.assert(WebReactComponent.register === register);
console.assert(WebReactComponent.Types === Types);
```